data/data/openstack/topology/sg-lb.tf

```diff
@@ -0,0 +1,61 @@
+resource "openstack_networking_secgroup_v2" "tnc" {
+  name = "tnc"
+}
+
+resource "openstack_networking_secgroup_rule_v2" "tnc_http" {
```

Comment:
TNC has been replaced by MCS (Machine Config Server) and this only listens on 49500 (HTTPS).

Reply:
Thanks for point this out.  I've fixed this.